instance/instance.go

```diff
@@ -32,9 +33,13 @@ const (
 
 //GetList a list of all instances
 func (s *Store) GetList(w http.ResponseWriter, r *http.Request) {
-	stateFilter := r.URL.Query().Get("state")
+	stateFilterQuery := r.URL.Query().Get("state")
+	var stateFilterList []string
+	if stateFilterQuery != "" {
+		stateFilterList = strings.Split(stateFilterQuery, ",")
```

Comment:
I would add a log statement here logging out the filter params.

Reply:
We currently don't do that logging on the import API:
https://github.com/ONSdigital/dp-import-api/blob/cmd-develop/api/api.go#L89

Do we want to be consistent between the two?

Reply:
You can leave this if you want as logging is a bigger issue than just this change / repo. Eleanor pointed out when we were trying to get CMD env running that logging was not very good so there is a plan to have a big talk about how to make it better. but for now ignore my comment